git-secrets-scan-action/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Scan with git-secrets'
+description: 'Scan repository with git-secrets'
+inputs:
+  path:
+    description: 'Relative path under $GITHUB_WORKSPACE to place the repository'
```

Comment:
This should say "Relative path under the current working directory or an absolute path to the repository that should be scanned".